openapi.yaml

```diff
@@ -16,175 +16,301 @@ host: api.oregonstate.edu
 basePath: /v1
 externalDocs:
   description: GitHub Repository
-  url: https://github.com/osu-mist/express-api-skeleton
+  url: https://github.com/osu-mist/notes-api
 produces:
   - application/json
 paths:
-  /pets:
+  /notes:
     get:
-      summary: List all pets
+      summary: Get notes for a student
       tags:
-        - pets
-      description: Get all pets
-      operationId: getPets
+        - notes
+      operationId: getNotes
       parameters:
-        - $ref: '#/parameters/authorization'
-        - $ref: '#/parameters/pageNumber'
-        - $ref: '#/parameters/pageSize'
-        - in: query
-          name: species
-          description: Search by pet species
+        - name: studentID
+          in: query
+          type: string
+          required: true
+          description: 9-digit OSU ID of student whose notes will be retrieved
+        - name: q
+          in: query
+          type: string
           required: false
+          description: Query string to be used when searching through the contents of each note
+        - name: sortKey
+          in: query
           type: string
+          required: false
+          default: 'dateModified'
```

Comment:
This was the value we used when designing the API, but I think it would make more sense to use `dateModified` since it's the name of a key in the note object. What are your thoughts on this?

Reply:
For this parameter, what other values can we use to sort the results? Can we put them into an `enum` object?